lib/src/configcat_client.dart

```diff
@@ -95,17 +95,17 @@ class ConfigCatClient {
       final result = await _getSettings();
       if (result.isEmpty) {
         final err =
-            'Config JSON is not present. Returning defaultValue: \'$defaultValue\'.';
-        _errorReporter.error(err);
+            'Config JSON is not present. Returning the `defaultValue` parameter that you specified in your application: \'$defaultValue\'.';
+        _errorReporter.error(1000, err);
         hooks.invokeFlagEvaluated(
             EvaluationDetails.makeError(key, defaultValue, err, evalUser));
         return defaultValue;
       }
       final setting = result.settings[key];
       if (setting == null) {
         final err =
-            'Value not found for key $key. Here are the available keys: ${result.settings.keys.join(', ')}';
-        _errorReporter.error(err);
+            'Failed to evaluate setting \'$key\' (the key was not found in config JSON). Returning the `defaultValue` parameter that you specified in your application: \'$defaultValue\'. Available keys: ${result.settings.keys.join(', ')}.';
```

Comment:
Available keys should be surrounded with single quotes like `'Key1', 'Key2', 'Key3'`